CHANGES.md

```diff
@@ -1,4 +1,189 @@
+## stable-2.3.0
+
+### Announcing Linkerd 2.3 🎈
+
+This stable release introduces a new TLS Identity system into the default
+Linkerd installation, replacing `tls=optional` and the `linkerd-ca` controller.
+Now, proxies generate ephemeral private keys into a tmpfs directory and
+dynamically refresh certificates, authenticated by Kubernetes ServiceAccount
+tokens, via the newly-introduced Identity controller.
+
+Now, all meshed HTTP communication is private and authenticated by default.
+
+In the CLI, this release includes 2 new commands:
+* `linkerd metrics` fetches metrics directly from a Linkerd proxy
+* `linkerd upgrade` upgrades your existing Linkerd installation
+
+Among the many improvements to the Web UI, we've added a Community page to
+surface news and updates from linkerd.io.
+
+For more details, see the announcement blog post:
+https://blog.linkerd.io/2019/04/16/announcing-linkerd-2-3/
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Upgrade notes**: The `linkerd-ca` controller is deprecated in favor of the
+`linkerd-identity` controller. If you had previously installed Linkerd with
+`tls=optional`, you will need to manually delete `linkerd-ca` after upgrading.
+Also, `single-namespace` mode is no longer supported. For full details on
+upgrading to this release, please see the
+[upgrade instructions](https://linkerd.io/2/upgrade/#upgrade-notice-stable-2-3-0).
```

Comment:
Right now when I go to this URL, I just get the "Upgrading Linkerd" page which starts with `Upgrade notice: stable-2.2.0` - will this be changing before this goes out?

Reply:
ah nice catch, i should have left a note that this URL, and the blog post, aren't live yet, will do that now...